README.md

```diff
@@ -11,31 +66,73 @@ This repository includes files with plain SQL that can be used to recreate the V
 - Check [queries.sql](./queries.sql) for examples of queries that can be run on a newly created database. **Important note: this file might include queries that make changes in the database (e.g., remove records). Use them responsibly!**
 
 
-## Authors
+### Prerequisites
+
+In order to run this project you need:
+
+- PostgreSQL installed
+- A GitHub Account
+- VS Code 
+
+### Setup
+
+- Open PSQL 
+- create a new database using CREATE DATABASE Vet Clinic
 
-👤 **Author1**
+### Usage
+
+- connect to database using the command \c Vet Clinic
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Stanley Nnamani AKA StarMindz**
 
 - GitHub: [@Starmindz](https://github.com/StarMindz)
-- LinkedIn: [Stanley Nnamani](https://www.linkedin.com/in/stanley-nnamani)
+- LinkedIn: [Stanley Nnamani](https://www.linkedin.com/in/stanley-nnamani/)
 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+<!-- CONTRIBUTING -->
 
-## 🤝 Contributing
+## 🤝 Contributing <a name="contributing"></a>
 
 Contributions, issues, and feature requests are welcome!
 
 Feel free to check the [issues page](../../issues/).
 
-## Show your support
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project, please give it a star and also follow me on GitHub for future projects
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+- Thumb ups to all the Naija Afrobeat Musicians whose beautiful music made building this project more fun
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
 
-Give a ⭐️ if you like this project!
+This project is [MIT](./LICENSE) licensed.
 
-## Acknowledgments
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## 📝 License
 
```

Comment:
### NOTE: I'm using this feature to add inline comments as I can't see any file changes in other files.

- Please, kindly add a range to the name, how many characters it should accept as leave the varchar without specifying is not ideal in maintaining your database [here](https://github.com/StarMindz/Vet-Clinic-/blob/a864f655da435dc2cf1d7367df2de9b46ccda8d4/schema.sql#L3-L11)

Reply:
- Please, let's also ensure we make the value for the weight a floating number since it's a decimal number to ensure we're sending the correct datatype to our database. see below:

```INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg)
VALUES(1, 'Agumon', '2020-02-03', 0, true, 10.23),
      (2, 'Gabumon', '2018-11-15', 2, true, 8),  // `8.0`
      (3, 'Pikachu', '2021-01-07', 1, false, 15.04),
      (4, 'Devimon', '2017-05-12', 5, true, 11); // `11.0`
```